pkg/model/components/apiserver.go

```diff
@@ -129,7 +129,11 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 	c.LogLevel = 2
 	c.SecurePort = 443
 
-	c.BindAddress = "0.0.0.0"
+	if clusterSpec.IsIPv6Only() {
```

Comment:
Any thoughts about using `IsIPv6CIDR`?
```suggestion
	if cidr.IsIPv6CIDR(clusterSpec.NonMasqueradeCIDR) {
```

Reply:
The fact that we are testing `NonMasqueradeCIDR` is not knowledge that should be spread all over the codebase. It should be centralized in one place.

Reply:
I don't think there will be that many places where this will be used, but I don't mind.
Maybe use `cidr.IsIPv6CIDR` inside `IsIPv6Only()`, to use the same approach everywhere in code?